.bowerrc

```diff
@@ -1,3 +1,3 @@
 {
-    "directory": "hu_web/hu_browser/static/hu_browser/bower/"
+    "directory": "hu_web/common-static/bower/"
 }
```

Comment:
Great choice of directory name here, better than what I was thinking.

Only note I have is that we might benefit for some namespacing here. (See the green "Static file namespacing" box on https://docs.djangoproject.com/en/1.11/howto/static-files/) Basically Django dumps all static files from all sources (static/ dirs in INSTALLED_APPS, and this directory) into one single static directory, so to avoid collisions, it's good practice to do namespacing like that link says.

Does that best/standard practice apply here, since this isn't an app? I don't know honestly, so I'm interested in your take on that